backEnd/prisma/schema.prisma

```diff
@@ -25,7 +24,6 @@ model MenuItem {
   description  String
   calorieCount Int
   imageUrl     String
-
```

Comment:
Is there a reason to remove the blank lines here? They were originally added to separate the FKs(foreign keys) from the standard columns, to help make this more readable.

Thanks for removing the commented out code!

Reply:
No reason, I guess I just find it more readable that way. I'll add them back.